libavformat/mxfdec.c

```diff
@@ -2068,8 +2196,172 @@ static MXFDescriptor* mxf_resolve_multidescriptor(MXFContext *mxf, MXFDescriptor
                 return sub_descriptor;
             }
         }
-    } else if (descriptor->type == Descriptor)
+    } else if (descriptor->type == Descriptor) {
+        // retrieve Soundfield Group identification
```

Comment:
Subdescriptors should be in their own `mxf_resolve_subdescriptor` function.

Reply:
it's just a revert, it's already present on mxfdec.c https://github.com/FFmpeg/FFmpeg/blob/master/libavformat/mxfdec.c#L2087